srcpkgs/monero/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'monero'
 pkgname=monero
-version=0.17.2.3
-revision=3
+version=0.18.1.2
+revision=1
 _randomx_version="1.1.9"
 # the revision monero uses as a submodule for the specific version
 _rapidjson_gitrev="129d19ba7f496df5e33658527a7158c79b99c21c"
```

Comment:
did you make sure submodule versions didn't change since the last 0.17.2.3?

Reply:
Monero uses randomx different version from release 1.1.9 with revision `261d58c77fc5547c0aa7fdfeb58421ba7e0e6e1c`.
rapidjson is the same.
supercop is the same.

Reply:
great, let's bump randomx to that revision then